try:
    3/0
except:
    print("Error Occurred")
